Src/Support/Google.Apis.Auth.AspNetCore/Google.Apis.Auth.AspNetCore.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <PropertyGroup>
+    <!-- Only ASP.NET Core 2.0 on netstandard2.0 supported -->
```

Comment:
This comment doesn't match the description. For example, presumably we're fine with ASP.NET Core 2.1?

Reply:
You can see I edited this file in a bit of a rush ;)
Fixed.